draft-ietf-cbor-time-tag.md

```diff
@@ -435,17 +463,22 @@ ClockQuality-group = (
 ### ClockClass (Key -2)
 
 Key -2 (ClockClass) can be used to indicate the clock class as per
-Table 5 of {{IEEE1588-2008}}.
+{{RFC8575}} (which is based on Table 5 in Section 7.6.2.4 of
+{{IEEE1588-2008}}; this has updated language as Table 4 in Section 7.6.2.5
+of {{IEEE1588-2019}}).
 It is defined as a one-byte unsigned integer as that is the range defined there.
 
 ### ClockAccuracy (Key -4)
 
 
-Key -4 (ClockAccuracy) can be used to indicate the clock accuracy as per
-Table 6 of {{IEEE1588-2008}}.
+Key -4 (ClockAccuracy) can be used to indicate the clock accuracy as
+per {{RFC8575}} (which is based on Table 6 in Section 7.6.2.5 of
+{{IEEE1588-2008}}; additional values have been defined in Table 5 in
+Section 7.6.2.6 of {{IEEE1588-2019}}).
 It is defined as a one-byte unsigned integer as that is the range defined there.
 The range between 32 and 47 is a slightly distorted logarithmic scale from
```

Comment:
With there being a specification for the range from 23 to 47, is there even a point in referencing the old scale that starts at 32?

Reply:
I think this is worth saying (in particular since RFC 8575 is based on -2008), but maybe just as a parenthesis to to the 2019 version presented as the full range.
(Not saying, together with the fact that 23 could look like a typo for 32, will create confusion.)
See commits below...